frontend/pages/Parts/[deviceHandle]/[itemTypeHandle].tsx

```diff
@@ -48,16 +48,11 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
          getGlobalSettings(),
```

Comment:
[...deviceHandle] there is a different syntax for matching multiple path elements. we could collapse this file and frontend/pages/Parts/[deviceHandle]/index.tsx into the same file.